scormxblock/static/js/src/scormxblock.js

```diff
@@ -124,7 +127,20 @@ function ScormXBlock_${block_id}(runtime, element) {
       }
       return false;
     }
+    function disableNextModuleArrow() {
+      var next_link = document.getElementsByClassName("next")[0];
+      var name = "complete-scorm future";
+      arr = next_link.className.split(" ");
```

Comment:
This is inefficient and error prone.
Please use jquery selector on line 131 and then `.addClass()` to simply add the class